exercise.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>The best me</title>
+
+    <!-- Bootstrap -->
+    <link href="css/bootstrap.min.css" rel="stylesheet">
+     <link rel="stylesheet" type="text/css" href="css/index.css"> -->
```

Comment:
This CSS file is unnecessary, you'll only need to modify js, and include the bootstrap.css and bootstrap-theme.css files.